sql/46_rsvps.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE gym
+    add `rsvps` TEXT default NULL;
```

Comment:
Why not `JSON` type? Since we put JSON in there, it's safer to use it as we have validation & ability to index virtual columns.

Reply:
Good question.
As background, in mariadb, JSON is an alias for LONGTEXT COLLATE utf8mb4_bin (with a validator check constraint)
All of the other json fields in golbat are defined as text.

Mariadb does not support the -> syntax, you have to use json_extract - which does not require the json type (as it doesn't exist)